fabric: update crd

vlktna opened the pull request:
CRD validation for the Fabric Gateway name: must not exceed 63 chars

for https://github.com/zalando-incubator/kubernetes-on-aws/pull/8759

AlexanderYastrebov commented on cluster/manifests/fabric-gateway/fabricgateway_crd.yaml:
```diff
@@ -456,6 +456,7 @@ spec:
                     description: |-
                       The name of the StackSet resource in the same namespace which
                       should be handling requests coming through FabricGateway
+                      maxLength: 63
```
Indentation does not look correct

vlktna committed:
fabric: update crd

CRD validation for the Fabric Gateway name: must not exceed 63 chars

for https://github.com/zalando-incubator/kubernetes-on-aws/pull/8759

AlexanderYastrebov commented:
:+1:

vlktna commented:
👍
